test/tests/postprocessors/scalar_coupled_postprocessor/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [./test]
+    type = 'Exodiff'
+    input = 'scalar_coupled_postprocessor_test.i’
```

Comment:
Nice work! The reason this failed the testing is because of that curly quote at the end of this line. If you'll fix that, squash, and re-push. I'll get this merged.